src/index.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './components/App';
```

Comment:
the App should always be directly in your src folder, not in the components folder